test/Tests.Infrastructure/NightlyBuildTheory.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using Raven.Client.Util;
+using Xunit;
+
+namespace Tests.Infrastructure
+{
+    public class NightlyBuildTheory : TheoryAttribute
+    {
+        public override string Skip
+        {
+            get
+            {
+                var now = SystemTime.UtcNow;
+                if (now.Hour >= 21 || now.Hour <= 6)
```

Comment:
That is a bad idea, I think.
I'm routinely running such tests at those hours.
I think it would be better to inspect an environment variable.